interface/ApfpInterfaceType.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+#include "Config.h"
+
+#ifdef APFP_GMP_INTERFACE_TYPE // Interface with GMP types
+#include <gmp.h>
+using ApfpInterfaceType = mpf_t;
```

Comment:
Get rid of all prefixes and put in namespace

Reply:
Agreed. We already rely on some C++ types so we can't `extern "C"` it even if we wanted to